operator/config/watcher/watcher.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: Watcher
+metadata:
+  name: controller-manager-watcher
+  namespace: system
+spec:
+  componentName: module-manager
+  contractVersion: v1
+  gvrsToWatch: [] # needs to be removed after removing it in the runtime-WatcherCRD
+  serviceInfo:
+    serviceName: event-service
+    servicePort: 2022
+
+---
+apiVersion: v1
```

Comment:
why not use the existing service in config/manager/manager.yaml?

Reply:
AS you can see in this PR I moved the Service from the manager.yaml to the watcher layer. So the service will just be deployed if the Watcher layer is activated. Since this service is just being used for the watched, I though it makes sense. I also did it the same way in the lifecycle manager

Reply:
let's follow up on this in another PR, if we want to keep one service or split (which IMO is not required)